### ISSUE TITLE
Lowercase projection identifiers?

### ISSUE THREAD

**Issue by Malvoz** [Maps4HTML/HTML-Map-Element#24]
In the spec, and in example usage elsewhere the `projection` attribute uses uppercase identifiers such as `projection="OSMTILE"`, while I understand the convention may be to spell it out that way, I think when it comes to HTML we should keep it lowercase. You rarely come across `crossorigin="ANONYMOUS"` or `type="VIDEO/MP4"` for example, in the wild.

**Comment by prushforth** [comment]
Yes. I can't remember why I made them uppercase, specifically. HTML is case-insensitive, but I couldn't find anything about attrubute values' cases sensitivity or not. I think I was trying to establish that the set of values is strictly limited and the meanings of those values is strictly defined by the spec. 

That said, I would prefer case-insensitive values. I'm sure my code will break somewhere currently, if fed a mixed or lower case value.

**Comment by prushforth** [comment]
I just came across a note in MDN that says that [DOMTokenLists are always case-sensitive](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList).  So this would affect how the projection attribute should be specified, I imagine, because it would be a [DOMTokenList](https://dom.spec.whatwg.org/#interface-domtokenlist) of size=== 1 ?
